### ISSUE TITLE
Petkit Fresh Element Gemini NOT SUPPORTED

### ISSUE THREAD

**Issue by rgreco87** [jubepue/homebridge-petkit-pet-feeder#7]
Please, add support for https://petkit.com/products/fresh-element-gemini (model "D4s").

I tried to get it work with your plugin but I always get the error message:

```
15/1/2023, 15:23:29 Petkit Bridge starting
15/1/2023, 15:23:41 Petkit Bridge Loaded plugin 'homebridge-petkit-feeder-fresh-element'
15/1/2023, 15:23:41 Petkit Bridge Loading 1 accessories...
15/1/2023, 15:23:41 Petkit Bridge WARNINGunhandled rejection: TypeError: Cannot read properties of undefined (reading 'DropMeal')
15/1/2023, 15:24:46 Petkit Bridge Configuration change
15/1/2023, 15:24:46 petkitbridge stopping
15/1/2023, 15:24:51 Petkit Bridge starting
15/1/2023, 15:24:59 Petkit Bridge Loaded plugin 'homebridge-petkit-feeder-fresh-element'
15/1/2023, 15:24:59 Petkit Bridge Loading 1 accessories...
15/1/2023, 15:27:36 petkitbridge stopping
15/1/2023, 15:38:02 petkitbridge stopping
```

**Comment by rgreco87** [comment]
```
Also having this issue:

16/1/2023, 10:55:09 Petkit Bridge starting
16/1/2023, 10:55:17 Petkit Bridge Loaded plugin 'homebridge-petkit-pet-feeder'
16/1/2023, 10:55:18 Petkit Bridge Loading 1 platforms...
16/1/2023, 10:55:18 Petkit Bridgepetkit_pet_feeder [INFO] Begin to initialize Petkit feeder platform.
16/1/2023, 10:55:18 Petkit Bridgepetkit_pet_feeder [INFO] Petkit feeder platform loaded.
16/1/2023, 10:55:18 Petkit Bridgepetkit_pet_feeder ERROR [ERROR] TypeError: Cannot read properties of undefined (reading 'forEach')
16/1/2023, 10:55:18 Petkit Bridge Bridge is running on port 51836.
```

**Comment by jubepue** [comment]
I have not abandoned it but for important reasons I have had to prioritize other things. The first week of February I start with it.

Jubepue

**Comment by rgreco87** [comment]
> I have not abandoned it but for important reasons I have had to prioritize other things. The first week of February I start with it.
> 
> Jubepue

Hello man,
Any news?

Thanks

**Comment by jubepue** [comment]
Hello!
Sorry for the delay but I was working on a new plugin to integrate all the petkit accessories in a single platform, it detects the accessories and creates, deletes and updates them automatically. You can install it on your homebridge by searching for "homebridge-petkit-platform". currently only works with feeder models but soon, eversweet and pura. https://github.com/jubepue/homebridge-petkit-platform

**Comment by imchesterhk** [comment]
> Hello! Sorry for the delay but I was working on a new plugin to integrate all the petkit accessories in a single platform, it detects the accessories and creates, deletes and updates them automatically. You can install it on your homebridge by searching for "homebridge-petkit-platform". currently only works with feeder models but soon, eversweet and pura. https://github.com/jubepue/homebridge-petkit-platform

thank you, so Fresh Element Gemini now being supported in "homebridge-petkit-platform" ?

**Comment by jubepue** [comment]
Yes!jubepueEl 14 abr 2023, a las 5:38, imchesterhk ***@***.***> escribió:﻿

Hello! Sorry for the delay but I was working on a new plugin to integrate all the petkit accessories in a single platform, it detects the accessories and creates, deletes and updates them automatically. You can install it on your homebridge by searching for "homebridge-petkit-platform". currently only works with feeder models but soon, eversweet and pura. https://github.com/jubepue/homebridge-petkit-platform

thank you, so Fresh Element Gemini now being supported in "homebridge-petkit-platform" ?
